core/src/main/java/com/netflix/msl/userauth/UserIdTokenAuthenticationFactory.java

```diff
@@ -37,9 +40,10 @@
      * 
      * @param authutils authentication utilities.
      */
-    public UserIdTokenAuthenticationFactory(final AuthenticationUtils authutils) {
+    public UserIdTokenAuthenticationFactory(final AuthenticationUtils authutils, final TokenFactory tokenFactory) {
```

Comment:
The TokenFactory should be accessed off the MslContext provided to authenticate() instead of being passed in as part of the ctor.